recaf-assembler/src/main/java/me/coley/recaf/assemble/ast/arch/MethodParameters.java

```diff
@@ -45,7 +45,7 @@ public List<MethodParameter> getParameters() {
 	public String print() {
 		return parameters.stream()
 				.map(MethodParameter::print)
-				.collect(Collectors.joining(", "));
+				.collect(Collectors.joining(","));
```

Comment:
Was removing the space needed here? The space IMO breaks up the parameters and makes it a bit more legible.

Reply:
yes because if not, you would need to have this at the end of the descriptor:
either: `.... I:v4 )` or `.... I:v4, )` or some weird parsing logic which will remove the last `)` and then add it back virtually in the `getDescriptor` method

Reply:
but i could make an extra MethodParamter group in my parser which basically consumes the `,` and and optionally takes a `)` at the end

Reply:
I think having the space would be nice if its not a lot of work :)

Reply:
i have been experimenting with syntax a bit and i think i can make it actually very dynamic by using some detection logic in the descriptor.

i imagined a syntax like this:
`compute(I v1, I v2, Ljava/lang/String; v3)I`

The not last elements are going to be easy just parse Identifier Identifier.remove(',') but the last element will have to contain the the `)` and the return descriptor. I Could move the descriptor to the end so that i can base end detection on the `)` but idk if:

`compute(I v1, I v2, Ljava/lang/String; v3) I`

is good.

Reply:
try to not change the syntax challenge (impossible)